packages/cli/src/platform/accelerate/enable.ts

```diff
@@ -69,11 +69,11 @@ export class Enable implements Command {
       return successMessage(
         `Accelerate enabled. Use this generated API key in your Accelerate connection string to authenticate requests: \n${generateConnectionString(
           payload.data.tenantAPIKey,
-        )}`,
+        )}\nFor more information, check out the Getting Started guide here: https://pris.ly/accelerate-getting-started`,
```

Comment:
```suggestion
        )}\nFor more information, check out the Getting Started guide here: https://pris.ly/d/accelerate-getting-started`,
```
to be changed in dub too, like mentioned in Slack